operator/pkg/cli/scheduler.go

```diff
@@ -71,6 +72,7 @@ func NewSchedulerClient(schedulerHost string, schedulerHostIsSet bool, authority
 		authority:     authority,
 		callOptions:   opts,
 		config:        config,
+		verbose:       verbose,
```

Comment:
:question: Is this actually used anywhere?  You're passing it in, but then CLI commands seem to still do their own printing, and `gopls` doesn't show this as being read

Reply:
will add back missing uses to show the request if verbose is true